chef-automate/README.md

```diff
@@ -1,260 +0,0 @@
-# Chef - Cloud Devops Solution Template
```

Comment:
Ok from the PR comments looks like we just want to remove this sample - but the diff looks like we're just emptying the files (not deleting them)???